### PR TITLE
Update README.md

### LINKED ISSUE #24: Add setup instructions
The project has no setup instructions in README.md

### PR DESCRIPTION
**Pull request by ramondelafuente**
This PR adds the basic installation instructions to the README.md

Closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+#### Run the following commands:
+
+- git clone [project]
+- composer install
+- php app/console doctrine:schema:update --force
+- php app/console fos:user:create
+
+#### To view the site, do:
+
+- app/console server run
```
Comment (dennisdegreef) [inline README.md]: `app/console server:run`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+
+- git
+- composer
+- a mysql database
+
+#### Run the following commands:
+
+- git clone [project]
+- composer install
+- php app/console doctrine:schema:update --force
```
Comment (dennisdegreef) [inline README.md]: Isn't it create for an initial run?

Comment (Joralf) [inline README.md]: This one: `php /app/console doctrine:database:create`

**Comment by dennisdegreef** [comment]
Any updates on this? :')

File: README.md (hunk 2 above)
Comment (ramondelafuente) [inline README.md]: `schema:update` will also create the tables if they don't exist.

**Comment by piwi91** [comment]
:+1:

**Comment by ramondelafuente** [comment]
Are all issues adressed or would you prefer `create` versus `update`?

**Pull request merged**